src/components/Calculator.js

```diff
@@ -1,49 +1,103 @@
 import React from 'react';
 import Touch from './Touch';
 import Display from './Display';
+import calculate from '../util/logic/calculate';
 
 export default class Calculator extends React.Component {
-  static renderTouch(i, cl) {
-    return <Touch value={i} className={cl} />;
+  constructor() {
+    super();
+    this.state = {
+      result: '0',
+      next: null,
+      operator: null,
+    };
   }
 
+  static renderTouch(props) {
+    return (
+      <Touch
+        onClick={() => props.onClick(props.value)}
+        value={props.value}
+        className={props.className}
+      />
+    );
+  }
+
+  // clear
+  clear = () => {
+    this.setState({
+      result: '0',
+      next: null,
+      operator: null,
+    });
+  };
+
+  clickHandler = (e) => {
+    this.setState((state) => calculate(state, e));
+  };
```

Comment:
- [ ] As the previous code reviewer requested Kindly add ÷ instead of one of the two pluses operations to match the design guidelines and requirements. Currently, you have two + signs in your calculator which doesn't make much sense.


## I didn't see that you changed it and used the "÷" instead of "+". 🔥🔥

Reply:
Sorry. It was just a matter of pushing